Add tooltip for affinity (#28434)
kubernetes/website: content/en/docs/reference/glossary/affinity.md

---

> title: affinity

Reviewer: nit:
```suggestion
title: Affinity
```

id: affinity
date: 2019-01-11

> full_link: /docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity/

Reviewer: ```suggestion
full_link: /docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
```

short_description: >

>   Node affinity is a set of rules used by the scheduler to determine where a pod can be placed.

Reviewer: ```suggestion
   Rules used by the scheduler to determine where to place pods.
```
1. The revised text is short (and should be).
2. There is more than one kind of affinity.

Author: Good to know :)

aka:
tags:

> - core object

Reviewer: I'm afraid that `core object` is incorrect; there is no `Affinity` object in the Kubernetes API. You should omit this line @mehabhalodiya

Author: Today I learned!

- fundamental
---

> Node affinity is a set of rules used by the scheduler to determine where a pod can be placed.

Reviewer: ```suggestion
In Kubernetes, _affinity_ is a set of rules that give hints to the scheduler about where to place pods.
```

<!--more-->

> 

Reviewer: ```suggestion
There are two kinds of affinity:

* [node affinity](/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity)
* [pod-to-pod affinity](/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity)

```

> The rules are defined using the familiar concepts of custom labels on {{< glossary_tooltip term_id="node" text="nodes">}} 

Reviewer: Because there are two kinds of affinity, I'd write:
```suggestion
The rules are defined using the Kubernetes {{< glossary_tooltip term_id="label" text="labels">}},
```
